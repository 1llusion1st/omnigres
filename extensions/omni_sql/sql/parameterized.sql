SELECT omni_sql.is_parameterized('SELECT 1'::omni_sql.statement);

SELECT omni_sql.is_parameterized('SELECT $1'::omni_sql.statement);