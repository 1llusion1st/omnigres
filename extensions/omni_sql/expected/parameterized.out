SELECT omni_sql.is_parameterized('SELECT 1'::omni_sql.statement);
 is_parameterized 
------------------
 f
(1 row)

SELECT omni_sql.is_parameterized('SELECT $1'::omni_sql.statement);
 is_parameterized 
------------------
 t
(1 row)

